### PR TITLE
[hugo] Revise the heading style (Demote headers)

### PR DESCRIPTION
**Pull request by eunchan**
Revise the HTML/CSS style to make the title to be a title of the page.
Other markdown headings are demoted in style (h1 tag --> h2 style).
And added h3 style.

This is related to PR #868

### CODE REVIEW

File: site/docs/layouts/_default/single.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
           {{ .TableOfContents }}
         </aside>
         <main id="main" role="main" class="markdown">
-          {{ if .Title }}<h1>{{ .Title }}</h1>{{ end }}
+          {{ if .Title }}<h1 class="title">{{ .Title }}</h1>{{ end }}
```
Comment (towoe) [inline site/docs/layouts/_default/single.html]: This should probably be duplicated in `list.html`

File: site/docs/assets/scss/_markdown.scss [hunk 2]
```diff
@@ -29,13 +25,26 @@
...
+  
+  h1.title {
+    font-size: 2.5em;
+    border: none !important;
+    font-family: 'Ubuntu', sans-serif !important;
+    font-weight: bold !important;
+  }
 
-  h3 {
+  h2 {
```
Comment (towoe) [inline site/docs/assets/scss/_markdown.scss]: Maybe h2 should set `font-family: 'Livvic', sans-serif;`

File: site/docs/layouts/_default/single.html (hunk 1 above)
Comment (gkelly) [reply site/docs/layouts/_default/single.html]: Yep, please do.  All `_index.md` files are rendered with `list.html`.

I think we should probably just make `list.html` a symlink to `single.html`.

Comment (eunchan) [reply site/docs/layouts/_default/single.html]: Revised and symlinked now.

File: site/docs/assets/scss/_markdown.scss (hunk 2 above)
Comment (eunchan) [reply site/docs/assets/scss/_markdown.scss]: h1 is previous h2. Current h1 has `'Livvic', sans-serif` style .

Comment (towoe) [reply site/docs/assets/scss/_markdown.scss]: Sorry, might be wrong here, this is just from observation not from css knowledge.
But previously h2 set the font-family, now is does not (same as h3) and they use the default from the body, which is Ubuntu. If that is the intention okay, but my assumption was that h? should use Livvic and only h1.title should use Ubuntu.

Comment (eunchan) [reply site/docs/assets/scss/_markdown.scss]: Now I understand. Thanks for the explanation. If that's an assumption in the site design, yeah it should have that font. Let me revise. :+1:

Comment (eunchan) [reply site/docs/assets/scss/_markdown.scss]: wait... Previous SCSS also doesn't have Livvic font in H2.

Comment (towoe) [reply site/docs/assets/scss/_markdown.scss]: Not sure if that is an assumption in the site design, but seemed like something to no change by mistake.
Well, h2 uses Livvic, but h3 does not.
Feels like we should have a clear understanding of how the site should look like, or rather if there are design rules which should be utilized.

Comment (eunchan) [reply site/docs/assets/scss/_markdown.scss]: This PR doesn't use Livvic for H2, and H3 both. Only H1 uses Livvic font family.
Current master branch uses Livvic for H2.

Demoting headers means `#` to be visible as `<H2>`, `##` to be visible as `<H3>` in the older version. So, I've changed CSS that H1 to have old H2 style, H2 to have old H3 style..

**Comment by eunchan** [comment]
@asb @sjgitty Could one of you review this PR and approve if it looks good?

**Comment by towoe** [comment]
Thanks @eunchan for updating the PR! The style does change, as is the intention of the PR, and results in a better look of the documentation.
I think there is still room for improvement in the overall header style appearance, which I will address in an issue so it can be discussed properly.
LGTM.

**Pull request merged**